CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
 ## VNext
+- Address vulnerability in `Newtonsoft.Json` ([GHSA-5crp-9r3c-p9vr](https://github.com/advisories/GHSA-5crp-9r3c-p9vr)). ([#2615](https://github.com/microsoft/ApplicationInsights-dotnet/pull/2615))
```

Comment:
be explicit that this Upgrade is only for Microsoft.ApplicationInsights.AspNetCore